src/lib.rs

```diff
@@ -172,6 +172,57 @@ struct Wallet {
     wallet_mutex: Mutex<BdkWallet<AnyDatabase>>,
 }
 
+pub struct OutPoint {
+    txid: String,
+    vout: u32,
+}
+
+pub struct TxOut {
+    value: u64,
+    address: String,
+}
+
+pub enum KeychainKind {
+    External = 0,
+    Internal = 1,
+}
+
+impl From<bdk::KeychainKind> for KeychainKind {
+    fn from(x: bdk::KeychainKind) -> KeychainKind {
+        match x {
+            bdk::KeychainKind::External => KeychainKind::External,
+            bdk::KeychainKind::Internal => KeychainKind::Internal,
+        }
+    }
+}
+
+pub struct LocalUtxo {
+    outpoint: OutPoint,
+    txout: TxOut,
+    keychain: KeychainKind,
+}
+
+trait NetworkLocalUtxo: {
+    fn from_utxo(x: &bdk::LocalUtxo, network: Network) -> LocalUtxo;
+}
+
+impl NetworkLocalUtxo for LocalUtxo {
+    fn from_utxo(x: &bdk::LocalUtxo, network: Network) -> LocalUtxo {
```

Comment:
I'd love it if we started commenting our codebase more (the FFI layer can be quite opaque and I want to make sure we make it easy on new contributors), and I think this is a perfect place to start. 

It took me a few minutes to understand what this `NetworkLocalUtxo` trait was for, and if I understand correctly, you added this trait because the `TxOut` struct now contains an `address` (`String`), and for that you need the [Address::from_script()](https://docs.rs/bitcoin/latest/bitcoin/util/address/struct.Address.html#method.from_script) which requires a network. That's great stuff and let me know if I didn't understand correctly. Long-winded way to say... I think we should just add a small comment explaining what this `NetworkLocalUtxo` is used for 🤣.

Reply:
I had some hard times finding a fitting and self-explanatory name for this trait, so I agree a comment could be useful here. For next contributions I will keep in mind that comments are welcome :smile: